docs/source/trainer/logging.rst

```diff
@@ -36,23 +36,25 @@ and also saves them to the file
     os.environ["WANDB_MODE"] = "disabled"
     os.environ["COMET_API_KEY"] = "<comet_api_key>"
     os.environ["MLFLOW_TRACKING_URI"] = ""
+    os.environ["NEPTUNE_MODE"] = "offline"
 
 .. testcode::
     :skipif: not _WANDB_INSTALLED or not _COMETML_INSTALLED
 
     from composer import Trainer
-    from composer.loggers import WandBLogger, CometMLLogger, MLFlowLogger, FileLogger
+    from composer.loggers import WandBLogger, CometMLLogger, MLFlowLogger, NeptuneLogger, FileLogger
 
     wandb_logger = WandBLogger()
     cometml_logger = CometMLLogger()
     mlflow_logger = MLFlowLogger()
+    neptune_logger = NeptuneLogger()
```

Comment:
Can we update the `README.md` as well? There is a section about integrations.

Reply:
You can also leave it to me. Loggers are mentioned in `docs/source/trainer/logging.rst` too. Let me know, so we don't get our wires crossed.

Reply:
@normandy7 I will then leave it in your more competent hands :rocket: :bow:

Reply:
I failed to note that this PR already includes changes to `logging.rst` 🤦🏻‍♀️ We're just missing a mention of neptune.ai on line 27, but I'm unable to commit/suggest directly.

```
Biases <https://www.wandb.com/>`__, `MLflow <https://www.mlflow.org/docs/latest/index.html>`__, `CometML <https://www.comet.com/?utm_source=mosaicml&utm_medium=partner&utm_campaign=mosaicml_comet_integration>`__, and `neptune.ai <https://neptune.ai/>`__,
```

Reply:
As for the README, updating line 108 to this should suffice:

```
- **Experiment tracking:** Weights and Biases, MLFlow, CometML, and neptune.ai — the choice is yours, easily log your data to your favorite platform.
```

I assumed there'd be more, but if the docstrings are now OK, as far as I can tell these neptune.ai mentions are all that's missing.

Reply:
Got it. The last commit should cover that. Could you confirm?

Reply:
Yep, looks good on my end.